Patch to run t-sdiv_qrnnd.c test with gcc-12.x compilers
See here:
https://github.com/flintlib/flint2/commit/fb8ddbcc06afa2a8f25637032fbb8211f6c6c122
https://github.com/flintlib/flint2/issues/1175
Author: J. Sassmannshausen (University College London/UK)
diff --git a/flint-2.9.0.orig/test/t-sdiv_qrnnd.c b/flint-2.9.0/test/t-sdiv_qrnnd.c
index 2e9822e..e2d65cd 100644
--- a/flint-2.9.0.orig/test/t-sdiv_qrnnd.c
+++ b/flint-2.9.0/test/t-sdiv_qrnnd.c
@@ -33,7 +33,7 @@ int main(void)
         {
             d = n_randtest_not_zero(state);
             nh = n_randtest(state);
-        } while ((FLINT_ABS(nh) >= FLINT_ABS(d)/2) || (nh == WORD_MIN));
+        } while ((nh == WORD_MIN) || (FLINT_ABS(nh) >= FLINT_ABS(d)/2)); 
 
         nl = n_randtest(state);
 
